[2025-05-14T16:45:15.710+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T16:40:00+00:00 [queued]>
[2025-05-14T16:45:15.723+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T16:40:00+00:00 [queued]>
[2025-05-14T16:45:15.724+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T16:45:15.725+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T16:45:15.725+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T16:45:15.741+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Load_Data> on 2025-05-14 16:40:00+00:00
[2025-05-14T16:45:15.747+0000] {standard_task_runner.py:52} INFO - Started process 36846 to run task
[2025-05-14T16:45:15.750+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Load_Data', 'scheduled__2025-05-14T16:40:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmp7vuynwsy', '--error-file', '/tmp/tmp5rjxx2zf']
[2025-05-14T16:45:15.752+0000] {standard_task_runner.py:80} INFO - Job 161: Subtask Load_Data
[2025-05-14T16:45:15.822+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T16:40:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T16:45:15.897+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Load_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T16:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T16:40:00+00:00
[2025-05-14T16:45:15.898+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T16:45:15.900+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Load_Data.py']
[2025-05-14T16:45:15.909+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T16:45:17.720+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-05-14T16:45:17.721+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-05-14T16:45:17.722+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-05-14T16:45:17.722+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-05-14T16:45:17.723+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-05-14T16:45:17.854+0000] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-14T16:45:17.917+0000] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-05-14T16:45:17.918+0000] {subprocess.py:92} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-05-14T16:45:17.920+0000] {subprocess.py:92} INFO - org.postgresql#postgresql added as a dependency
[2025-05-14T16:45:17.921+0000] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-ba197ebe-ec8c-44fe-a083-d844e0a5e4f5;1.0
[2025-05-14T16:45:17.922+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T16:45:18.018+0000] {subprocess.py:92} INFO - 	found org.postgresql#postgresql;42.6.0 in central
[2025-05-14T16:45:18.044+0000] {subprocess.py:92} INFO - 	found org.checkerframework#checker-qual;3.31.0 in central
[2025-05-14T16:45:18.056+0000] {subprocess.py:92} INFO - :: resolution report :: resolve 132ms :: artifacts dl 2ms
[2025-05-14T16:45:18.057+0000] {subprocess.py:92} INFO - 	:: modules in use:
[2025-05-14T16:45:18.057+0000] {subprocess.py:92} INFO - 	org.checkerframework#checker-qual;3.31.0 from central in [default]
[2025-05-14T16:45:18.058+0000] {subprocess.py:92} INFO - 	org.postgresql#postgresql;42.6.0 from central in [default]
[2025-05-14T16:45:18.059+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T16:45:18.060+0000] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-14T16:45:18.060+0000] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-14T16:45:18.061+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T16:45:18.061+0000] {subprocess.py:92} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-05-14T16:45:18.062+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T16:45:18.063+0000] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-ba197ebe-ec8c-44fe-a083-d844e0a5e4f5
[2025-05-14T16:45:18.063+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T16:45:18.064+0000] {subprocess.py:92} INFO - 	0 artifacts copied, 2 already retrieved (0kB/4ms)
[2025-05-14T16:45:18.222+0000] {subprocess.py:92} INFO - 25/05/14 16:45:18 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-14T16:45:18.423+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-05-14T16:45:18.424+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-05-14T16:45:18.425+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-05-14T16:45:29.355+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T16:45:29.406+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Load_Data, execution_date=20250514T164000, start_date=20250514T164515, end_date=20250514T164529
[2025-05-14T16:45:29.433+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T16:45:29.479+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
